util/dir/src/lib.rs

```diff
@@ -28,6 +28,8 @@ use bigint::hash::{H64, H256};
 use journaldb::Algorithm;
 use helpers::{replace_home, replace_home_and_local};
 use app_dirs::{AppInfo, get_app_root, AppDataType};
+// re-export platform-specific functions
+use platform::*;
 
 #[cfg(target_os = "macos")] const AUTHOR: &'static str = "Parity";
 #[cfg(target_os = "macos")] const PRODUCT: &'static str = "io.parity.ethereum";
```

Comment:
Sorry to grumble again, would you mind moving this constants to `platform` as well?